src/GeneralStateTestsFiller/stCreateTest/CREATE_EOF1DeployValidOpcodesFiller.yml

```diff
@@ -0,0 +1,82 @@
+# Deploy every Valid Opcode (not including PUSH opcodes)
+CREATE_EOF1DeployValidOpcodes:
+  env:
+    currentCoinbase: 2adc25665018aa1fe0e6bc666dac8fc2697ff9ba
+    currentDifficulty: '0x020000'
+    currentGasLimit: '89128960'
+    currentBaseFee: '10'
+    currentNumber: '1'
+    currentTimestamp: '1000'
+    previousHash: 5e20a0453cecd065ea59c37ac63e079ee08998b6045136a8ce6635c7912ec0b6
+
+  pre:
+    a94f5374fce5edbc8e2a8697c15331677e6ebf0b:
+      balance: 1000000000
+      code: ''
+      nonce: 0
+      storage: {}
+    b94f5374fce5edbc8e2a8697c15331677e6ebf0b:
+      balance: 0
+      code: |
+        :yul
+        {
+          // valid opcodes
+          mstore(0, 0x000102030405060708090a0b101112131415161718191a1b1c1d203031323334)
+          mstore(32, 0x35363738393a3b3c3d3e3f404142434445464748505152535455565758595a5b)
+          mstore(64, 0x808182838485868788898a8b8c8d8e8f909192939495969798999a9b9c9d9e9f)
+          mstore(96, 0xa0a1a2a3a4f0f1f2f3f4f5fafdfeff0000000000000000000000000000000000)
+
+          // initcode: { mstore(0, 0xef0001010002000000) return(23, 9) } }
+          mstore(128, 0x68ef000101000200000060005260096017f30000000000000000000000000000)
+          for { let os := 0 } lt(os, 111) { os := add(os, 1) } 
+          { 
+            let code := shr(248, mload(os))
+            mstore8(136, code) // change returned byte in initcode
+            if iszero(create(0, 128, 19)) { sstore(code, 1) }
+          }
+          sstore(256, 1)
+        }
+      nonce: 0
+      storage: {}
+
+  transaction:
+    data: 
+      - ''
+    gasLimit:
+      - 89000000
+    gasPrice: 10
+    nonce: 0
+    secretKey: 45a915e4d060149eb4365960e6a7a45f334393093061116b197e3240065ff2d8
+    to: 'b94f5374fce5edbc8e2a8697c15331677e6ebf0b'
+    value:
+    - 0
+
+  expect:
+    - indexes:
+       data: !!int -1
+       gas: !!int -1
+       value: !!int -1
+      network:
+        - 'London+3540+3670'
+      result:
+       a94f5374fce5edbc8e2a8697c15331677e6ebf0b:
+          nonce: 1
+       b94f5374fce5edbc8e2a8697c15331677e6ebf0b:
+          nonce: 111
+          storage:
+            '256': 1
+       f1ecf98489fa9ed60a664fc4998db699cfa39d40:
+         nonce: 1
+         code: '0xef0001010002000000'
+       30c7cc0d18123b445c2636ff9069ef28c0dc32b5:
+         nonce: 1
+         code: '0xef0001010002000100'
+       e86DeAED2AAB4645FE1e1FC01A58A0ef8637Bb80:
+         nonce: 1
+         code: '0xef000101000200fe00'
+       e86DeAED2AAB4645FE1e1FC01A58A0ef8637Bb80:
```

Comment:
Is this the same address as just above? How come it has different code expectation?

Reply:
Good catch, I have changed to the correct address. Interesting that retesteth does not complaint about this.

Reply:
Consider reporting to retesteth.